src/config-schema.ts

```diff
@@ -239,7 +226,171 @@ export const configSchema = {
+            target: '[data-extension-id="patient-lists-dashboard-link"]',
+            content:
+              'Creating a patient list is a great way to keep a group of patients organized for easier management. Click here to go to the patient lists view',
+            disableBeacon: true,
+            spotlightClicks: true,
+            hideNextButton: true,
+            data: {
+              autoNextOn: '[title="All lists"]',
+            },
+          },
+          {
+            target: '[title="All lists"]',
+            content: 'Click here to see all the patient lists we have in the system.',
+            hideNextButton: true,
+            hideBackButton: true,
+            spotlightClicks: true,
+            spotlightPadding: 20,
+            data: {
+              autoNextOn: '[data-testid="patientListsTable"]',
+            },
+          },
+          {
+            target: '[data-openmrs-role="New List"]',
+            title: 'New List',
+            content: "Now, let's try creating a new list. Click here to open the new patient list form.",
+            spotlightClicks: true,
+            placement: 'left',
+            hideNextButton: true,
+            hideBackButton: true,
+            data: {
+              autoNextOn: '[data-openmrs-role="Patient List Form"]',
+            },
+          },
+          {
+            target: '[data-openmrs-role="Patient List Form"]',
+            title: 'Patient List Form',
+            content: 'Enter the necessary details and click on "Create List" to create the list.',
+            spotlightClicks: true,
+
+            hideBackButton: true,
+            hideNextButton: true,
+            placement: 'left',
+            data: {
+              autoNextOn: '.omrs-snackbars-container div',
+            },
+          },
+          {
+            target: '[data-testid="patientListsTable"]',
+            title: 'Patient Lists table',
+            content:
+              'Great! We have successfully created a new patient list. Find and click on the patient list you created to proceed. You can use the search option at the top right if needed.',
+            disableOverlay: true,
+            spotlightClicks: true,
+            spotlightPadding: 40,
+            hideBackButton: true,
+            hideNextButton: true,
+            data: {
+              autoNextOn: '[data-openmrs-role="Patient Empty tile"]',
+            },
+          },
+          {
+            target: '[data-openmrs-role="Patient Empty tile"]',
+            content: "You can see the patient list is empty since we haven't added any patients to the list yet.",
+            hideBackButton: true,
+            disableOverlay: true,
+          },
+          {
+            target: '[data-testid="searchPatientIcon"]',
+            title: 'Search icon',
+            content:
+              'To access the patient chart, first, you need to select a patient. To start, click here to open the search box so that we can search for a patient.',
```

Comment:
```suggestion
              'To add a patient to the list, we have to go to the respective patient’s chart view. Click here to open the search box so that we can search for a patient.',
```